oak/server/oak_grpc_node.cc

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2018 The Project Oak Authors
```

Comment:
```suggestion
 * Copyright 2019 The Project Oak Authors
```

Reply:
I think 2018 is OK if the file is mostly-copied from an existing 2018 file?

Reply:
Done